tests/ntp.bats

```diff
@@ -10,3 +20,8 @@ load test-helper
   run "${clicmd}" ntp local
```

Comment:
This test is inside of the `ntp internet` test when it should be its own test.

I passed with a false positive despite errors from running `treehouses ntp local` manually:
![image](https://user-images.githubusercontent.com/61807745/115817144-1dd47800-a3af-11eb-9e3f-4d3073040e3f.png)

Reply:
I'm not sure I'm understanding you correctly. You're saying that ntp local should be in its own test case?
because that's how it is in the commit
![ntplocaltest](https://user-images.githubusercontent.com/23529006/115928748-a0fbd980-a454-11eb-8ef1-c52370823942.jpg)